src/mapml/control/AttributionControl.js

```diff
@@ -0,0 +1,17 @@
+export var attributionControl = function (map) {
+    map._attributionControl = map._map.attributionControl.setPrefix('<button onclick="this.closest(\'.leaflet-container\').querySelector(\'.shortcuts-dialog\').showModal()" class="shortcuts-button mapml-button">Keyboard shortcuts</button> | <a href="https://www.w3.org/community/maps4html/" title="W3C Maps for HTML Community Group">Maps4HTML</a> | <img src="data:image/svg+xml;base64,PHN2ZyBhcmlhLWhpZGRlbj0idHJ1ZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB3aWR0aD0iMTIiIGhlaWdodD0iOCI+PHBhdGggZmlsbD0iIzRDN0JFMSIgZD0iTTAgMGgxMnY0SDB6Ii8+PHBhdGggZmlsbD0iI0ZGRDUwMCIgZD0iTTAgNGgxMnYzSDB6Ii8+PHBhdGggZmlsbD0iI0UwQkMwMCIgZD0iTTAgN2gxMnYxSDB6Ii8+PC9zdmc+" style="padding-right: 0.3em;" alt="Slava Ukraini"> <a href="https://leafletjs.com" title="A JS library for interactive maps">Leaflet</a>');
+
+    let dialog = document.createElement("dialog");
+    dialog.setAttribute("class", "shortcuts-dialog");
+    dialog.setAttribute("autofocus", "");
+    dialog.onclick = function (e) {
+        e.stopPropagation();
+    };
+    dialog.innerHTML = `<b>${M.options.locale.kbdShortcuts} </b><button onclick='this.parentElement.close()'>X</button>` +
```

Comment:
```suggestion
    dialog.innerHTML = `<b>${M.options.locale.kbdShortcuts} </b><button aria-label="Close" onclick='this.parentElement.close()'>X</button>` +
```

The close button is missing an accessible name. See [MDN: Interactive elements must be labeled](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names#interactive_elements_must_be_labeled) for more info.

(I suppose "Close" should be translated as well at some point, per https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/149.)